assets/js/controllers/modal_notification_ctrl.js

```diff
@@ -0,0 +1,4 @@
+walletApp.controller("ModalNotificationCtrl", ($scope, Wallet, $modalInstance, notification) => {
+  $scope.notification = notification;
+  $scope.ok = () => $modalInstance.close(notification);
```

Comment:
I prefer to have this function enclosed with brackets and on it's own line (without a return statement).